manifoldco/ui
# Add field to set API token in docs

## Reason for change



Make it easy to update the token in the docs so that people don't have to go into the console to set the token.

## Testing


- Go to docs and set token
- check local storage to see that token is set correctly
- ensure examples fetch with token
